App Crash on Login

**Summary:** 

Login leads to crash of app

**Steps to reproduce:** 

Start App 
Type in login data.
Hit "login" button. 

**System logs:**

```

STACK_TRACE=java.lang.RuntimeException: Unable to start activity ComponentInfo{fr.free.nrw.commons/fr.free.nrw.commons.contributions.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.accounts.Account.hashCode()' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2678)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2743)
	at android.app.ActivityThread.-wrap12(ActivityThread.java)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1490)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6165)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:888)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:778)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int android.accounts.Account.hashCode()' on a null object reference
	at android.os.Parcel.readException(Parcel.java:1690)
	at android.os.Parcel.readException(Parcel.java:1637)
	at android.content.IContentService$Stub$Proxy.setSyncAutomaticallyAsUser(IContentService.java:1060)
	at android.content.ContentResolver.setSyncAutomaticallyAsUser(ContentResolver.java:2107)
	at android.content.ContentResolver.setSyncAutomatically(ContentResolver.java:2097)
	at fr.free.nrw.commons.contributions.MainActivity.initMain(MainActivity.java:110)
	at fr.free.nrw.commons.contributions.MainActivity.onCreate(MainActivity.java:84)
	at android.app.Activity.performCreate(Activity.java:6687)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1140)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2631)
	... 9 more



```

**Device and Android version:** 

ANDROID_VERSION=7.1.1
PHONE_MODEL=Moto G Play
 
**Commons app version:** 

APP_VERSION_CODE=561
APP_VERSION_NAME=2.12.3


**Would you like to work on the issue?**

I don't have the qualification to work on this issue

--- a/app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java
+++ b/app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java
@@ -21,6 +21,7 @@ import androidx.fragment.app.FragmentManager.OnBackStackChangedListener;
 import androidx.fragment.app.FragmentTransaction;
 
 import fr.free.nrw.commons.MediaDataExtractor;
+import fr.free.nrw.commons.auth.SessionManager;
 import io.reactivex.disposables.Disposable;
 import java.util.List;
 
@@ -94,6 +95,9 @@ public class ContributionsFragment
 
     @Inject ContributionsPresenter contributionsPresenter;
 
+    @Inject
+    SessionManager sessionManager;
+
     private LatLng curLatLng;
 
     private boolean firstLocationUpdate = true;
@@ -162,7 +166,8 @@ public class ContributionsFragment
             showContributionsListFragment();
         }
 
-        if (!ConfigUtils.isBetaFlavour()) {
+        if (!ConfigUtils.isBetaFlavour() && sessionManager.isUserLoggedIn()
+            && sessionManager.getCurrentAccount() != null) {
             setUploadCount();
         }
 

